Constitution.txt

```diff
@@ -1,490 +1,264 @@
+2: Every Bill which shall provide for the funding of actions of the Federal Government of these United States prescribed by this Constitution which has been presented first to the House of Representatives, received the approval and signature of two-thirds of their number, presented to the Senate, and received the approval and signature of two-thirds of their number, shall be presented to the President for his approval. If he should approve the Bill by signature, it shall become Law; if not, or if he shall not provide his approval within 14 calendar days of the Senate's approval, it shall be returned to the House of Representatives, and if three-fourths of the Representatives approve, it shall become law. Bills passed into law shall retain force of law for a period to be defined in the Bill, but not to exceed six years.
 
-3: Every Order, Resolution, or Vote to which the Concurrence of the Senate and House of Representatives may be necessary (except on a question of Adjournment) shall be presented to the President of the United States; and before the Same shall take Effect, shall be approved by him, or being disapproved by him, shall be repassed by two thirds of the Senate and House of Representatives, according to the Rules and Limitations prescribed in the Case of a Bill.
+3: Every other Bill which has been presented to the House of Representatives and the Senate, and has been approved and signed by all Representatives and Senators from several of the States, shall be sent to the Chief Executive of each of those States. If he should approve the Bill, it shall be considered approved by his State; if not, he shall send it to the Legislature of his State. If, after consideration, the Legislature of the State approves and signs the Bill in keeping with that State's Constitution, it shall be considered approved by the State. When at least two States have approved the Bill, it shall become law in those states which have so approved it, for a period not to exceed six years from first approval.
```

Comment:
This section essentially allows for States to form internal treaties under the structure of the Union, without necessarily involving every State in a given internal treaty. For example, the states of California, Washington, New York, and Massachusetts could instantiate a cross-state universal healthcare provision. That would allow the citizens of those states to all share a policy which they appear all to want, and take advantage of economies of scale, using their own budgets, without imposing the policy on states like Idaho, Wyoming, and Indiana, where such policies are not popular.

This process requires total consent within each participating State, but doesn't require approval from any non-participant. It bears more similarities to a business contract than to a political struggle, and this should help to preserve the Union even in the face of significant policy disagreements.